app/controllers/projects_controller.rb

```diff
@@ -1,21 +1,22 @@
 class ProjectsController < ApplicationController
   def index
-    @projects = current_user.organization&.projects
-    redirect_to new_project_path unless @projects.present?
+    redirect_to new_organization_project_path(organization) unless projects.present?
```

Comment:
This will fail if `projects` is empty. It will try to redirect to `new_organization_project_path(organization)` but `organization` returns `nil`